introduction.md

```diff
@@ -1 +1,2 @@
-My name is Jesus. JavaScript and Git are my favourite paths
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?